src/utils/Draw.ts

```diff
@@ -0,0 +1,84 @@
+import { DependencyLink, DependencyNode, Network } from '../components/types';
+import {
+    createSimulation,
+    createSVGContainer,
+    createZoom,
+    generateLabelPath,
+    generateMarkers,
+    generateLinkPath,
+    getLabelTextHeight,
+    handleDrag,
+} from './helpers/GraphHelpers';
+
+export const draw = (network: Network, container: HTMLDivElement) => {
+    const { nodes, links } = network;
+
+    const width = container.clientWidth + 500;
+    const height = container.clientHeight + 500;
```

Comment:
Why 500 pixels?

Reply:
Actually right now its some random value which seemed to fit. Need to investigate what value is best and work on it.